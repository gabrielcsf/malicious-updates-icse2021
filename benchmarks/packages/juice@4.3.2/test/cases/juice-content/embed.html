<html>
<head>
    <link rel="stylesheet" type="text/css" href="embed.css" data-embed />
</head>
<body>
    <p>hi</p>
</body>
</html>
