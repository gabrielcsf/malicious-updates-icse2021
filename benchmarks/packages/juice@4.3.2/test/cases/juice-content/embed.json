{
    "url": "./test/cases/juice-content",
    "removeStyleTags": true
}