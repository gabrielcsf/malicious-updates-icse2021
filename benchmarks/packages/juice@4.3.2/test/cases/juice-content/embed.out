<html>
<head>
    <style type="text/css">
p {
    color: blue;
}
</style>
</head>
<body>
    <p>hi</p>
</body>
</html>
